data-otservbr-global/scripts/talkactions/player/aol.lua

```diff
@@ -1,14 +1,15 @@
 local aol = TalkAction("!aol")
 
-function aol.onSay(player, words, param)	
-    if player:removeMoneyBank(50000) then
-        player:getPosition():sendMagicEffect(CONST_ME_MAGIC_BLUE)
-        player:addItem(3057, 1)    
-        player:sendTextMessage(MESSAGE_EVENT_ADVANCE, "You have bought an amulet of loss for 50K!")
-    else
-        player:getPosition():sendMagicEffect(CONST_ME_POFF)
-        player:sendCancelMessage("You do not have enough money.")
-    end
+function aol.onSay(player, words, param)
+	if player:removeMoneyBank(50000) then
+		player:getPosition():sendMagicEffect(CONST_ME_MAGIC_BLUE)
+		player:addItem(3057, 1)
+		player:sendTextMessage(MESSAGE_EVENT_ADVANCE, "You have bought an amulet of loss for 50K!")
+	else
+		player:getPosition():sendMagicEffect(CONST_ME_POFF)
+		player:sendCancelMessage("You do not have enough money.")
+	end
 end
 
+aol:groupType("normal")
```

Comment:
should this just be the default?

Reply:
I thought about this, but I thought it would get in the way in case someone forgets to set the parameter. I put a report in the "register" informing that the "groupType" parameter was forgotten.